imod/schemata.py

```diff
@@ -503,6 +503,21 @@ def validate(self, obj: GridDataArray, **kwargs) -> None:
             )
 
 
+class NUniqueValueSchema(BaseSchema):
+    """
+    Schema to validate if amount of unique values is correct.
```

Comment:
It appears that the validate method reports an error when the number of unique values exceeds the indicated amount ("length"), but not when it has less unique values than the indicated amount. Please modify the docstring to stress that.

Reply:
Good one, will improve!